framework/src/postprocessors/Receiver.C

```diff
@@ -37,18 +39,22 @@ Receiver::Receiver(const InputParameters & params)
   write_data.needReporterTimeIndex<PostprocessorValue>(r_name, 2);
 
   // Initialize values
-  const Real & value = getParam<Real>("default");
-  write_data.setReporterValue<PostprocessorValue>(r_name, value, 0);
+  write_data.setReporterValue<PostprocessorValue>(r_name, _first_value, 0);
   if (_initialize_old)
   {
-    write_data.setReporterValue<PostprocessorValue>(r_name, value, 1);
-    write_data.setReporterValue<PostprocessorValue>(r_name, value, 2);
+    write_data.setReporterValue<PostprocessorValue>(r_name, _first_value, 1);
+    write_data.setReporterValue<PostprocessorValue>(r_name, _first_value, 2);
   }
 }
 
 Real
 Receiver::getValue()
 {
-  // Return the stored value (references stored value in getPostprocessorData)
-  return _my_value;
+  // if the current value is the same as the default, then we assume we are changing
+  // value according to an external actor. Otherwise, we assume we are changing
+  // value according to the default (which is controllable)
+  if (MooseUtils::absoluteFuzzyEqual(_default, _first_value))
+    return _my_value;
+  else
+    return _default;
```

Comment:
Hmm this just doesn't feel right to me. Someone more familiar with reporters, @loganharbour @zachmprince, how would you guys go about what @aprilnovak wants to do in #24247 ?

Reply:
I just want to control a postprocessor which just holds a value - I do agree this feels weird and could have conflict resolution issues if mistakenly used a `MultiAppPostprocessorTransfer` to write into the postprocessor, while _also_ controlling it. I could do a `ConstantPostprocessor` instead, lmk

Reply:
A `ConstantPostprocessor` sounds really good to me